src/main/java/org/isf/medicalinventory/model/MedicalInventory.java

```diff
@@ -74,7 +74,19 @@ public class MedicalInventory extends Auditable<String> {
 
 	@Column(name = "MINVT_WRD_ID_A")
 	private String ward;
+	
+	@Column(name = "MINVT_CHARGE_TYPE")
+	private String chargeType;
+	
+	@Column(name = "MINVT_DISCHARGE_TYPE")
+	private String dischargeType;
+	
+	@Column(name = "MINVT_SUPPLIER")
+	private Integer supplier;
```

Comment:
why is this an `Integer`?  Can it be null?  If not then I think it should be an `int`

Reply:
yes It can be null